src/settings/Settings.ts

```diff
@@ -402,6 +402,10 @@ export const SETTINGS: {[setting: string]: ISetting} = {
         supportedLevels: LEVELS_DEVICE_ONLY_SETTINGS_WITH_CONFIG,
         default: "en",
     },
+    "spell-check-languages": {
```

Comment:
Electron [docs](https://www.electronjs.org/docs/api/session#sesgetspellcheckerlanguages) suggest it persists the spell checker selection itself in some mystical storage location. Would it perhaps be better to add a get path that reads from its own storage, instead of our own setting that duplicate Electron's storage and could diverge?

(Perhaps the setting has advantages I am missing, so I am just wondering which is best.)